### PR TITLE
Update app.js -> Fix 404 links

### PR DESCRIPTION
**Pull request by sebastian-marinescu**
Fixed some dead links.

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -5,7 +5,7 @@
 window.addEventListener('DOMContentLoaded', function() {
 
   // We'll ask the browser to use strict code to help us catch errors earlier.
-  // https://developer.mozilla.org/Web/JavaScript/Reference/Functions_and_function_scope/Strict_mode
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode
```
Comment (tofumatt) [inline js/app.js]: The locale shouldn't be included in the URLs.

**Comment by tofumatt** [comment]
The locale shouldn't be included in the URLs... any chance you could remove them for this patch? Otherwise r+; looks good!

Thanks! :smile:

**Comment by sebastian-marinescu** [comment]
Check, it's done.

**Comment by tofumatt** [comment]
Awesome, thanks! :+1:

**Pull request merged**